src/groundlight/client.py

```diff
@@ -75,6 +80,12 @@ def __init__(self, endpoint: Optional[str] = None, api_token: Optional[str] = No
                         If unset, fallback to the environment variable "GROUNDLIGHT_API_TOKEN".
         :type api_token: str
 
+        :param disable_tls_verification: Set this to false to skip verifying SSL/TLS certificate
+                                        when calling API from https server. In unset, fallback to
+                                        the environment variable "TLS_VERIFY". By default, certificates
+                                        are verified.
```

Comment:
```suggestion
        :param disable_tls_verification: Set this to `True` to skip verifying SSL/TLS certificates
                                        when calling API from https server. In unset, fallback it will check
                                        the environment variable "TLS_VERIFY" for `1` or `0`. By default, 
                                        certificates are verified.

                                        Disable verification if using a self-signed TLS certificate with a Groundlight 
                                        Edge Endpoint.  It is unadvised tto disable verification  if connecting directly 
                                        to the Groundlight cloud service. 
```